plugins/woocommerce/changelog/update-woocommerce-blocks-10.4.0

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: update
+
+Update WooCommerce Blocks to 10.4.0
```

Comment:
I'm thinking that maybe that files is unnecessary considering that update to 10.4.0 was never merged, so technically we're updating from 10.2.3 to 10.4.2 directly, which is covered in `plugins/woocommerce/changelog/update-woocommerce-blocks-10.4.2`.

But that's my assumption, I'm not 100% of the process here.

Reply:
I understand your concern. However, I feel the changelog should be there.